### PR TITLE
Set test double for helpers 

### PR DESCRIPTION
**Pull request by shineability**
- Added `setHelperTestDouble()` method
- Update README.md
- Small PSR-2 fix

### CODE REVIEW

**Comment by shineability** [comment]
@Vinai Looks ok to you?

File: lib/DigitalPianism/TestFramework/Model/Config.php [hunk 1]
```diff
@@ -12,6 +12,20 @@ class DigitalPianism_TestFramework_Model_Config extends Mage_Core_Model_Config
      */
     private $resourceModelTestDoubles = [];
 
+    /**
+     * Set test double for helper class.
+     *
+     * @param $helperClass
+     * @param $testDouble
```
Comment (Vinai) [inline lib/DigitalPianism/TestFramework/Model/Config.php]: The argument types are missing in the docblock `@param` annotations.

**Comment by Vinai** [comment]
Very nice, thanks! I just had one tiny comment regarding the DocBlock.
I'm happy with it, but I'll leave it to @digitalpianism to do the merge as it's his repo.

**Comment by shineability** [comment]
@Vinai Great, tnx!

@digitalpianism Argument types are also missing for the `setModelResourceTestDouble` and `setModelTestDouble`methods. Maybe add them for all `set*TestDouble` methods when merging?

**Comment by shineability** [comment]
@digitalpianism Any thoughts on this? :)

**Comment by Vinai** [comment]
@digitalpianism seems to be busy, I'll go ahead and merge myself.
Do you want to add the missing type annotations for the existing methods to the PR?
That would be great, but if not, I'll merge this PR the way it is.

**Comment by shineability** [comment]
@Vinai Tnx, I'll add the annotations...

**Comment by shineability** [comment]
@Vinai Done.

**Comment by Vinai** [comment]
Thank you!

**Comment by digitalpianism** [comment]
Sorry yeah I'm sooooo busy ATM thanks for your update I'll trust Vinai on that, he's the testing boss

**Comment by Vinai** [comment]
Miss having you around the usual social channels @digitalpianism

**Pull request merged**